apps/src/applab/levels.js

```diff
@@ -87,6 +89,10 @@ levels.ec_simple = {
       'color': 'yellow',
       'blocks': []
     },
+    'Storage': {
+      'color': 'orange',
```

Comment:
did orange just work without having to add additional styles? I think I had to add some for yellow in common.scss because droplet didn't already handle that color in the palette.